receiver/awsxrayreceiver/go.mod

```diff
@@ -7,6 +7,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/commo
 require (
 	github.com/aws/aws-sdk-go v1.34.8
 	github.com/google/uuid v1.1.1
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/common v0.0.0-00010101000000-000000000000
```

Comment:
Need to have a replace with local here

Reply:
oh that was done. It's on ln5. Will try to split common/aws in its own package